docker/crons.conf

```diff
@@ -0,0 +1,2 @@
+* * * * * /bin/bash /usr/bin/restart.sh
+* * * * * echo "Hello world" >> /usr/src/app/phantomjs.log
```

Comment:
Nope, it is every minute. 

There is a command `* * * * * echo "Hello world" >> /usr/src/app/phantomjs.log `which will put `Hello world` into a file every minute. So with that command you will see that the cron actually works. The log will show each time cron has executed something